cos-fleet-catalog-kamelets/kafka/src/main/resources/kamelets/cos-kafka-source.kamelet.yaml

```diff
@@ -50,6 +52,7 @@ spec:
         keyDeserializer: "{{keyDeserializer:org.apache.kafka.common.serialization.ByteArrayDeserializer}}"
         valueDeserializer: "{{valueDeserializer:org.bf2.cos.connector.camel.serdes.bytes.ByteArrayDeserializer}}"
         kafkaClientFactory: "#{{kafka-client-factory}}"
+        headerSerializer: "#{{kafka-header-serializer}}"
```

Comment:
is thi correct ? since this is a source, there should not be the need to serialize headers

Reply:
This is to override the default header serializer as a workaround for CAMEL-18380, which is fixed in camel 3.19
https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/pull/530/files#diff-8b4a55735e98e651635642ad00f6f9b7fbab6745f276bcb3d91144d0f5be9920R11

Reply:
yes but it is needed ? since it is a source, the header serialization should not be invoked. 
I don't think it would harm if we leave it but should not have any effect (unless I missed something)

Reply:
Oh OK that sounds right, I'll remove it, thanks!

Reply:
Done